optimize/client/src/modules/components/index.js

```diff
@@ -51,3 +51,4 @@ export {InstanceViewTable} from './InstanceViewTable';
 export {TenantInfo} from './TenantInfo';
 export {TargetSelection} from './TargetSelection';
 export {Loading} from './Loading';
+export {Breadcrumbs} from './Breadcrumbs';
```

Comment:
Thanks for sharing. I did not know about this problem.
It seems that it is not a problem for esbuild as long as we do not use "*" in imports.
For development, it will slow it down, as vite currently loads everything in barrel files. Vite is however considering improving this: https://github.com/vitejs/vite/issues/8237

I won't create an issue for this for now since we already have a lot of existing migrations we are doing. I am happy to revisit this in the future after we merge the apps. It might be good to align with other webapps on this anyway.